tests/test_secure_random.py

```diff
@@ -77,13 +84,15 @@ def test_invalid_args(self):
 class TestSeed(unittest.TestCase):
     with tf.Session():
         s = seed_gen()
+        s0 = seed_gen()
 
         minval = -2000
         maxval = 0
 
         shape = [2, 3]
 
         output = seeded_random_uniform(shape, seed=s, minval=minval, maxval=maxval).eval()
+        output2 = seeded_random_uniform(shape, seed=s0, minval=minval, maxval=maxval).eval()
```

Comment:
is `output2` ever used?

Reply:
just leftover from an old thing I was testing, will remove!